### PR TITLE
chore(docs): add a getting started guide

### PR DESCRIPTION
**Pull request by michelleN**
This PR applies to #296.

### CODE REVIEW

File: docs/usage_docs/getting-started-guide.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#Getting Started Guide
+
+[Helm](https://helm.sh) helps you find and use software built for Kubernetes. With a few Helm commands you can quickly and easily deploy software packages like:
+
+- Postgres
+- etcd
+- HAProxy
+- redis
+
+All of the Helm charts live at [github.com/kubernetes/charts](https://github.com/kubernetes/charts). If you want to make your own charts we have a guide for [authoring charts](authoring_charts.md). Charts should follow the defined [chart format](/docs/deisgn/chart_format.md).
```
Comment (vaikas-google) [inline docs/usage_docs/getting-started-guide.md]: deisgn -> design in the chart format link.

File: docs/usage_docs/getting-started-guide.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Get started with the following steps:
+
+1. [Clone](github.com/kubernetes/helm) this project.
+
+2. Then, run `make build`. This will install all the binaries you need in `bin/` in the root of the directory.
+
+3. To start playing with helm, run `./bin/helm`
+
+4. `helm dm install` to install the server side component.
```
Comment (vaikas-google) [inline docs/usage_docs/getting-started-guide.md]: wouldn't it be better to install dm before asking them to play with it?

File: docs/usage_docs/getting-started-guide.md (hunk 1 above)
Comment (michelleN) [inline docs/usage_docs/getting-started-guide.md]: oo0o good catch

File: docs/usage_docs/getting-started-guide.md (hunk 2 above)
Comment (michelleN) [inline docs/usage_docs/getting-started-guide.md]: Yeah, I guess that's a bit confusing. I think I was trying to say "To see a list of helm commands, run `./bin/helm`". Does that sound better to you?

**Pull request merged**